pkg/service/model_build_load_balancer.go

```diff
@@ -232,42 +232,94 @@ func (t *defaultModelBuildTask) resolveLoadBalancerSubnets(ctx context.Context,
 }
 
 func (t *defaultModelBuildTask) buildLoadBalancerAttributes(_ context.Context) ([]elbv2model.LoadBalancerAttribute, error) {
-	var attrs []elbv2model.LoadBalancerAttribute
-	accessLogEnabled := t.defaultAccessLogS3Enabled
-	bucketName := t.defaultAccessLogsS3Bucket
-	bucketPrefix := t.defaultAccessLogsS3Prefix
-	if _, err := t.annotationParser.ParseBoolAnnotation(annotations.SvcLBSuffixAccessLogEnabled, &accessLogEnabled, t.service.Annotations); err != nil {
+	defaultAttribs := t.getDefaultAnnotationLbAttributes()
+	configMapAttribs, err := t.getConfigMapLbAttributes()
+	if err != nil {
 		return []elbv2model.LoadBalancerAttribute{}, err
 	}
-	if accessLogEnabled {
-		t.annotationParser.ParseStringAnnotation(annotations.SvcLBSuffixAccessLogS3BucketName, &bucketName, t.service.Annotations)
-		t.annotationParser.ParseStringAnnotation(annotations.SvcLBSuffixAccessLogS3BucketPrefix, &bucketPrefix, t.service.Annotations)
-	}
-	crossZoneEnabled := t.defaultLoadBalancingCrossZoneEnabled
-	if _, err := t.annotationParser.ParseBoolAnnotation(annotations.SvcLBSuffixCrossZoneLoadBalancingEnabled, &crossZoneEnabled, t.service.Annotations); err != nil {
+	specificAttribs, err := t.getAnnotationSpecificLbAttributes()
+	if err != nil {
 		return []elbv2model.LoadBalancerAttribute{}, err
 	}
+	mergedAttribs := mergeAttributeMaps(defaultAttribs, mergeAttributeMaps(configMapAttribs, specificAttribs))
```

Comment:
lets use the existing algorithm.MergeStringMap which takes in multiple maps in order of precedence from high to low.